README.md

```diff
@@ -1,6 +1,10 @@
 # Installation
 
     git clone https://github.com/tswr/.vimrc.git ~/vim-tswr/
+    bash ~/vim-tswr/setup.sh
+
+The file setup.sh contains following instructions:
```

Comment:
These comments are unnecessary, it's github, anyone can look inside of setup.sh :)